bib/internal/setup/setup.go

```diff
@@ -53,6 +53,14 @@ func EnsureEnvironment() error {
 	if err := util.RunCmdSync("mount", "--bind", destPath, osbuildPath); err != nil {
 		return err
 	}
+
+	// Ensure we have devfs inside the container to get dynamic loop
+	// loop devices inside the container.
+	devMnt := "/dev/"
```

Comment:
minor nit: Why the separate variable here?  It doesn't seem useful?

Reply:
Yeah, silly me, fixed :)